app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  helper_method :favorites
+
+  def favorites
```

Comment:
accessible by all views via all controllers, allows application.html.erb to access favorites information in the session